src/components/Project/ProjectBasics/ProjectForm/ProjectForm.tsx

```diff
@@ -39,27 +48,48 @@ const ProjectForm = () => {
     when: isDirty,
   });
 
+  const [newProjectId, setNewProjectId] = useState<string>('');
+
+  useEffect(() => {
```

Comment:
Add a descriptive comment to useEffect

Reply:
Is this still needed or could the navigation be added to onSubmit() finally clause?

Reply:
Still needed since we tried this before